src/Enhavo/Bundle/AppBundle/Resources/public/js/classes/Admin.js

```diff
@@ -398,15 +398,14 @@ function Admin (router, templating, translator)
       var id = element.parents('[data-sortable-row]').data('id');
       var url;
       if (directionUp) {
-        url = moveUpUrl;
+        url = router.generate(moveUpRoute, {id: id});
       } else {
-        url = moveDownUrl;
+        url = router.generate(moveDownRoute, {id: id});
```

Comment:
If the route stays constant, why generate the url every call instead of once in init?